### PR TITLE
Smoke test

### PR DESCRIPTION
**Pull request by nmaslarski**

### CODE REVIEW

File: lib/cf/broker/manifest.yml [hunk 1]
```diff
@@ -7,11 +7,11 @@ applications:
   disk_quota: 512M
   env:
     CONF: default
-    DEBUG: e-abacus-*
+    DEBUG: metering-*,e-metering-*
```
Comment (hsiliev) [inline lib/cf/broker/manifest.yml]: I think we need to have `e-abacus-*` as well to support error debugs for used modules.

Comment (hsiliev) [inline lib/cf/broker/manifest.yml]: We should rename the debug string to `abacus-broker` to match the rest of the abacus modules.

File: test/broker/src/test/broker-smoke-test.js [hunk 2]
```diff
@@ -0,0 +1,190 @@
...
+            clientId = credentials.client_id;
+            expect(credentials.client_secret).to.not.equal(null);
+            clientSecret = credentials.client_secret;
+            expect(credentials.collector_url)
+              .to.contain('abacus-usage-collector');
+            expect(credentials.dashboard_url).to.not.equal(null);
+            done();
+          });
+        });
+      context('and posting usage', () => {
```
Comment (hsiliev) [inline test/broker/src/test/broker-smoke-test.js]: New line here, to make the next context more visible?

File: lib/cf/broker/get-usage.sh [hunk 3]
```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+set -e
+
+CLIENT_ID=abacus
+CLIENT_SECRET='s3cret'
```
Comment (hsiliev) [inline lib/cf/broker/get-usage.sh]: Do we need this script as part of the broker? We might move it to another helper repo.

The proper place for scripts is in `bin` dir of the project. If these scripts are used for Concourse we might place them in `etc/concourse/scripts`

File: lib/cf/broker/post-usage.sh [hunk 4]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash
+
+set -e
+
+instance_id='8b97d2da-9710-49ae-89d9-5a3773811592'
```
Comment (hsiliev) [inline lib/cf/broker/post-usage.sh]: Do we need the script in this repo or can we move it to another aux one?

File: lib/cf/broker/post-usage.sh [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+CLIENT_ID="abacus-rp-${instance_id}"
+CLIENT_SECRET='Huijeu6MM4'
```
Comment (hsiliev) [inline lib/cf/broker/post-usage.sh]: Hard-coding the secret does not seem like a good idea for two reasons:
* not all installations will have the same secret
* security

File: test/broker/src/test/env.sh [hunk 6]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```
Comment (astefanova) [inline test/broker/src/test/env.sh]: We don't need this file?

**Pull request merged**